src/sentry/integrations/github/tasks/pr_comment.py

```diff
@@ -224,9 +224,9 @@ def github_comment_workflow(pullrequest_id: int, project_id: int):
             {
                 "name": f"Root cause #{i + 1}",
                 "type": "copilot-chat",
-                "prompt": f"@sentry root cause issue {str(issue_id)} with PR URL https://github.com/{repo.name}/pull/{str(pullrequest_id)}",
```

Comment:
Is this currently broken in prod?

Reply:
It doesn't break anything on the Sentry side. Our logic in Seer just wants a different number. And this isn't available to any customers.